server/boot/a-extendUser.js

```diff
@@ -77,7 +77,7 @@ module.exports = function(app) {
       from: 'Team@freecodecamp.com',
       subject: 'Welcome to freeCodeCamp!',
       protocol: isDev ? null : 'https',
-      host: isDev ? 'localhost' : 'freecodecamp.com',
+      host: isDev ? (process.env.HOST || 'localhost') : 'freecodecamp.com',
```

Comment:
Not a big fan of additional conditionals in a ternary. Let's break this up into it's own const's and then use the const in the ternary